server/src/test/resources/config/extensions.yml

```diff
@@ -3,11 +3,11 @@ extensions:
       ephemeralId: uniqueid1
```

Comment:
This isn't part of this PR, but isn't the `ephemeralId` auto-assigned, and should this be the (unique) persistent id?